rest/model/admin.go

```diff
@@ -412,12 +413,6 @@ func (a *APIAuthConfig) BuildFromService(h interface{}) error {
 				return errors.Wrap(err, "could not build API Okta auth settings from service")
 			}
 		}
-		if v.Okta != nil {
```

Comment:
Why did you delete this?

Reply:
This is a copy-paste of the lines above this (but GitHub hides too much of the surrounding context to see it from the file preview).